appsec-kit-v7/src/main/java/com/vaadin/appsec/v7/ui/content/VulnerabilitiesTab.java

```diff
@@ -23,7 +23,6 @@
 import com.vaadin.ui.Alignment;
 import com.vaadin.ui.Button;
 import com.vaadin.ui.ComboBox;
-import com.vaadin.ui.Component;
 import com.vaadin.ui.Grid;
 
```

Comment:
Adding and applying the Vaadin analysis [filter](https://github.com/vaadin/appsec-kit/pull/57/files#diff-56fc99d4edfa92f6651818ba8764c2dab220c33ead71e3cde7cd6bf44336057fR53) is not complete yet though it is shown on the UI.

The same applies for v8 version.

Reply:
I removed the Vaadin analysis filter, let's add it later once we have some data for it.